docs/RELEASING.md

```diff
@@ -0,0 +1,11 @@
+## Releasing npm package
+
+* Install or upgrade `np` package globally (`npm install -g np`)
+* Login to npm with `npm adduser`
+* Create new branch, e.g. `1.2.0`, add upstream
+* Run this to publish package
+
+```
+yarn build
```

Comment:
Should we execute this when having `prepack` doing the same?

Reply:
We don't need it actually. I will remove it, but I'll add a note about the importance of building before publishing